frontend/views/pages/Contributions.vue

```diff
@@ -69,23 +69,86 @@ page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle')
       @cancel='handleIncomeCancel'
     )
 
+  page-section(title='Debug Income Details')
+    form(
+      novalidate
+      @submit.prevent='setPaymentInfo'
+    )
+      .field
+        i18n.label(
+          tag='label'
+          :args='{ mincome: groupSettings.mincomeAmount, currency }'
+        ) Do you make at least {currency}{mincome} per month?
+
+        .radio-wrapper
+          input.radio(
+            type='radio'
+            value='incomeAmount'
+            v-model='form.incomeDetailsKey'
+            id='radio-receiving'
+          )
+          //- TODO: update design system to make radio buttons use labels instead of spans (and update _forms.scss accordingly)
+          i18n(tag='label' for='radio-receiving') No, I don't
```

Comment:
There's a more correct way to group related input radio/checkboxes, in a A11Y way.
Here's a dumb example for you can adapt:

```pug
fieldset // this groups everything
  legend What's your favorite food?
  label
    input(type="radio" name="apple")
    Apple
  label
    input(type="radio" name="bananas")
    Bananas
 label
    input(type="radio" name="kiwis")
    Kiwis
```

With this:
- All inputs are accessible related thanks to `fieldset`. The "question" for this groupd should be placed under `legend`.
- Pro tip: By placing `input` inside `label` we don't need to worry about using `for` and `id`. It just works for any modern screen reader.

You can read more about this subject [in this a11y style guide]https://a11y-style-guide.com/style-guide/section-forms.html).

If CSS doesn't look great (it won't), don't worry much about it, next I can make it flawless ✨